src/tooltip.js

```diff
@@ -24,5 +24,5 @@ export function createTooltipHTML({ name, groupNumber, nodeMeasure }) {
   tooltip.appendChild(groupNumberEntry);
   tooltip.appendChild(nodeMeasureEntry);
 
-  return tooltip.innerHTML;
```

Comment:
is any unit test cover it?

Reply:
No. for this repo there is no test cases previously written.

Reply:
No. for this repo there is no test cases previously written.

Reply:
I suggest that unit tests should be added as we don't know the reason it behaves differently and @Caele also recommends doing that as https://github.com/qlik-oss/sn-network-chart/blob/master/src/sn-paint.js#L56

Reply:
There is no test cases are written particular for this “sn-network-chart” repo, so can you please suggest me how and where to write test cases.

Reply:
I have added test cases.

Reply:
Nice! Great work